zeebe/exporters/rdbms-exporter/src/main/java/io/camunda/exporter/rdbms/FormExportHandler.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.exporter.rdbms;
+
+import io.camunda.db.rdbms.write.domain.FormDbModel;
+import io.camunda.db.rdbms.write.domain.FormDbModel.FormDbModelBuilder;
+import io.camunda.db.rdbms.write.service.FormWriter;
+import io.camunda.zeebe.protocol.record.Record;
+import io.camunda.zeebe.protocol.record.intent.FormIntent;
+import io.camunda.zeebe.protocol.record.value.deployment.Form;
+import java.nio.charset.StandardCharsets;
+
+public class FormExportHandler implements RdbmsExportHandler<Form> {
+
+  private final FormWriter formWriter;
+
+  public FormExportHandler(final FormWriter formWriter) {
+    this.formWriter = formWriter;
+  }
+
+  @Override
+  public boolean canExport(final Record<Form> record) {
+    return record.getIntent() instanceof FormIntent;
+  }
+
+  @Override
+  public void export(final Record<Form> record) {
+    if (record.getIntent().equals(FormIntent.CREATED)) {
+      formWriter.create(map(record, false));
+    } else if (record.getIntent().equals(FormIntent.DELETED)) {
+      formWriter.update(map(record, true));
+    }
+  }
+
+  private FormDbModel map(final Record<Form> record, final boolean isDeleted) {
+    final var value = record.getValue();
+    return new FormDbModelBuilder()
+        .formKey(value.getFormKey())
+        .formId(value.getFormId())
+        .tenantId(value.getTenantId())
+        .schema(new String(value.getResource(), StandardCharsets.UTF_8))
```

Comment:
💭  For BPMN and DMN files also the resource name is stored here and also part of the API Entity. Do we maybe need to align something here?

Reply:
I did it like it was done in the importer (and the new exporter). The FormEntity also doesn't contain a resource name.